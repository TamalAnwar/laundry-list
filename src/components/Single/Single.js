import React from 'react';
import PropTypes from 'prop-types';

const Single = (props) => {
  return <div className="single">Single Cloth Page</div>;
};

Single.propTypes = {
  item: PropTypes.object
};

export default Single;
