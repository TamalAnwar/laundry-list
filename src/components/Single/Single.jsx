import React from 'react';
import PropTypes from 'prop-types';

const Single = ({ name }) => {
  return <div className="single">{name}</div>;
};

Single.propTypes = {
  name: PropTypes.string.isRequired
};

export default Single;
