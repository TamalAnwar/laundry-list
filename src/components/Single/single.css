.single {
  border: 2px solid purple;
  border-radius: 10px;
  height: 100px;
  text-align: center;
  background-size: cover;
}
