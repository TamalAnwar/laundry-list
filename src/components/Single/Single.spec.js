import React from 'react';
import { shallow } from 'enzyme';

import Single from './Single';

describe('<Single/>', () => {
  it('should render correctly in debug mode', () => {
    const component = shallow(<Single debug />);
    expect(component).toMatchSnapshot();
  });
});
