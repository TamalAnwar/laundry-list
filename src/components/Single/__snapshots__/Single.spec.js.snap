// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Single/> should render correctly in debug mode 1`] = `
<div
  className="single"
/>
`;
